enterprise/server/raft/store/store.go

```diff
@@ -567,6 +564,9 @@ func (s *Store) Start() error {
 
 func (s *Store) Stop(ctx context.Context) error {
 	s.log.Info("Store: started to shut down")
+	if s.driverQueue != nil {
```

Comment:
Why do we need these if not nil checks in start and stop?

Reply:
because we have the flag to enable driver. if the flag is off, the driverQueue is nil and can cause nil pointer excepter.